### PR TITLE
[WIP] Adding user presistent UI settings

### PR DESCRIPTION
**Pull request by SMaxOwok**

### CODE REVIEW

File: client/src/store/reducers/ui/persistent/reader.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
-import { combineReducers } from "redux";
+// import { combineReducers } from "redux";
 import colors from "./reader/colors";
 import typography from "./reader/typography";
+import { handleActions } from "redux-actions";
 
-export default combineReducers({
+const initialState = {
```
Comment (SMaxOwok) [inline client/src/store/reducers/ui/persistent/reader.js]: This is where I'm struggling right now.  I want the `initialState` of `reader` to be in the shape of `colors` and `typography` which have their own `initialState`s defined.

**Comment by zdavis** [comment]
See ed3f36f. Isn't this what you want? I think this is what the initial state should look like. The reader module just combines the colors and typography reducers; it doesn't provide any state of its own. With this commit, initial state looks like this:

image

**Comment by SMaxOwok** [comment]
@zdavis this is almost working now (though obviously needs some refactoring).  The only actions that aren't triggering the user to update right now are the `incrementAttribute` and `decrementAttribute` functions.

**Comment by SMaxOwok** [comment]
Closing in favor of a fresh PR

**Pull request closed without merging**